packages/amplify-provider-awscloudformation/src/push-resources.js

```diff
@@ -480,6 +524,26 @@ function formNestedStack(context, projectDetails, categoryName, resourceName, se
           Object.assign(parameters, { env: currentEnv });
         }
 
+        // If auth is imported check the parameters section of the nested template
+        // and if it has auth or unauth role arn or name, then inject it from the
+        // imported auth resource's properties
+        const { imported, authRoleArn, authRoleName, unauthRoleArn, unauthRoleName } = context.amplify.getImportedAuthRoles(context);
```

Comment:
Like I mentioned in the CLI package can we move the getImportedAuthRole to this package

Reply:
See my response earlier.